src/modules/iol2opc/include/module.h

```diff
@@ -324,6 +329,10 @@ class IOL2OPCBridge : public RFModule, public iol2opc_IDL
      */
     CvPoint getBlobCOG(const Bottle &blobs, const int i);
 
+    bool    getBlobPoints(const CvPoint &cog, deque<CvPoint> &blobPoints);
```

Comment:
Done for this [here](https://github.com/robotology/icub-client/commit/ba2dd3273568f984df62f437a7e4e5d7d9a4a014)